src/Compatibility/Microsoft.DotNet.Compatibility.Console/Program.cs

```diff
@@ -0,0 +1,63 @@
+﻿// Copyright (c) .NET Foundation and contributors. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using Microsoft.DotNet.Compatibility.Console;
+using Microsoft.DotNet.PackageValidation;
+
+if (args.Length != 11)
+{
+    throw new ArgumentException("Pass in the set of required parameters...");
```

Comment:
It's unclear to me what the usage of this CLI tool looks like based on just the args below.
Could we add a usage message?

Reply:
Fixed. The console app now uses System.CommandLine for parsing the args.